python/adbc_driver_manager/adbc_driver_manager/dbapi.py

```diff
@@ -721,10 +733,17 @@ def execute(self, operation: Union[bytes, str], parameters=None) -> None:
             The query to execute.  Pass SQL queries as strings,
             (serialized) Substrait plans as bytes.
         parameters
-            Parameters to bind.  Can be a Python sequence (to provide
-            a single set of parameters), or an Arrow record batch,
-            table, or record batch reader (to provide multiple
-            parameters, which will each be bound in turn).
+            Parameters to bind.  Can be a Python sequence (to bind a single
+            set of parameters), a Python dictionary (to bind a single set of
```

Comment:
Does providing a dictionary automatically set the option to bind by name? If not, should it?

Reply:
It will, yes. It's only when using Arrow data that you have to manage it implicitly.